content/blog/2017-11-16-ropensci-changes.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: "ropensci-changes"
+title: "changes: easy Git-based version control from R"
+date: 2017-11-16
+authors:
+  - name: Anikó B. Tóth (with edits from Nick Golding)
+categories: blog
+topicid: ###
+tags:
+  - community
+  - ropensci
+  - meetings
+  - unconf
+  - australia
+  - r
+  - Git
+  - versioncontrol
+---
+
+Are you new to version control and always running into trouble with Git?
+Or are you a seasoned user, haunted by the traumas of learning Git and reliving them whilst trying to teach it to others?
+Yeah, us too.
+
+Git is a version control tool designed for software development, and it is immensely powerful. It didn’t actually dawn on me quite how amazing Git is until I spent a weekend in Melbourne with a group of Git whizzes using Git to write a package targeted toward Git beginners. 
```

Comment:
Good to refer here to ozunconf, maybe with a link to Nick Tierney's post https://ropensci.org/blog/2017/10/31/ozunconf2017/, because later, when you say "After browsing through the available project ideas..." it's not clear what you are referring to.

Reply:
I see the nice paragraph about ozunconf near the end, so just give enough context for the reader to understand the scenario